ui/src/app/settings/containers/Network/Network.js

```diff
@@ -1,5 +1,33 @@
 import React from "react";
+import { useSelector } from "react-redux";
 
-const Network = () => <h4>Network</h4>;
+import selectors from "app/settings/selectors";
+import Loader from "app/base/components/Loader";
+import ProxyForm from "app/settings/components/ProxyForm";
+
+const Network = () => {
+  const loaded = useSelector(selectors.config.loaded);
+  const loading = useSelector(selectors.config.loading);
+
+  const httpProxy = useSelector(selectors.config.httpProxy);
+  const proxyType = useSelector(selectors.config.proxyType);
```

Comment:
Please don't pass state down to children, just call `useSelector` in the child. This is particularly good in the hooks world as state is now encapsulated (and coupled...) in a component which means any props a component defined are actual props, not redux state.

Reply:
@squidsoup Ahh right, I've still got the muscle memory to pass down props from one parent connected component. In that case maybe we _should_ stop with container/component naming and maybe instead go with a views/routes structure instead, which imo is much easier to traverse since it's reflective of the DOM. (sidenote - what's an easy way to generate a gh markdown file directory?)
```bash
...
├── views/
│   ├── Images/
│   ├── Network/
│   │   ├── NetworkContainer/
│   │   │   ├── index.js
│   │   │   └── NetworkContainer.js
│   │   ├── ProxyForm/
│   │   │   ├── index.js
│   │   │   └── ProxyForm.js
│   │   ├── DNSForm/
│   │   │   ├── index.js
│   │   │   └── DNSForm.js
...
```